Help: when "cmake  --build . --config Release" ,    error: unknown type name 'FINDEX_INFO_LEVELS'

cmake --build . --config Release
[  1%] Building C object CMakeFiles/ggml.dir/ggml.c.obj
In file included from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/winbase.h:18,
                 from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/windows.h:70,
                 from D:\wuwei\LLM\LLAMA\llama.cpp\ggml.c:43:
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/fileapi.h:168:65: error: unknown type name 'FINDEX_INFO_LEVELS'; did you mean 'GET_FILEEX_INFO_LEVELS'?
  168 |   WINBASEAPI HANDLE WINAPI FindFirstFileExA (LPCSTR lpFileName, FINDEX_INFO_LEVELS fInfoLevelId, LPVOID lpFindFileData, FINDEX_SEARCH_OPS fSearchOp, LPVOID lpSearchFilter, DWORD dwAdditionalFlags);
      |                                                                 ^~~~~~~~~~~~~~~~~~
      |                                                                 GET_FILEEX_INFO_LEVELS
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/fileapi.h:168:121: error: unknown type name 'FINDEX_SEARCH_OPS'
  168 |   WINBASEAPI HANDLE WINAPI FindFirstFileExA (LPCSTR lpFileName, FINDEX_INFO_LEVELS fInfoLevelId, LPVOID lpFindFileData, FINDEX_SEARCH_OPS fSearchOp, LPVOID lpSearchFilter, DWORD dwAdditionalFlags);
      |
        ^~~~~~~~~~~~~~~~~
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/fileapi.h:169:66: error: unknown type name 'FINDEX_INFO_LEVELS'; did you mean 'GET_FILEEX_INFO_LEVELS'?
  169 |   WINBASEAPI HANDLE WINAPI FindFirstFileExW (LPCWSTR lpFileName, FINDEX_INFO_LEVELS fInfoLevelId, LPVOID lpFindFileData, FINDEX_SEARCH_OPS fSearchOp, LPVOID lpSearchFilter, DWORD dwAdditionalFlags);
      |                                                                  ^~~~~~~~~~~~~~~~~~
      |                                                                  GET_FILEEX_INFO_LEVELS
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/fileapi.h:169:122: error: unknown type name 'FINDEX_SEARCH_OPS'
  169 |   WINBASEAPI HANDLE WINAPI FindFirstFileExW (LPCWSTR lpFileName, FINDEX_INFO_LEVELS fInfoLevelId, LPVOID lpFindFileData, FINDEX_SEARCH_OPS fSearchOp, LPVOID lpSearchFilter, DWORD dwAdditionalFlags);
      |
         ^~~~~~~~~~~~~~~~~
In file included from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/windows.h:76:
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/winreg.h:200:59: error: unknown type name 'PVALENTA'
  200 |   WINADVAPI LONG WINAPI RegQueryMultipleValuesA(HKEY hKey,PVALENTA val_list,DWORD num_vals,LPSTR lpValueBuf,LPDWORD ldwTotsize);
      |                                                           ^~~~~~~~
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/winreg.h:201:59: error: unknown type name 'PVALENTW'
  201 |   WINADVAPI LONG WINAPI RegQueryMultipleValuesW(HKEY hKey,PVALENTW val_list,DWORD num_vals,LPWSTR lpValueBuf,LPDWORD ldwTotsize);
      |                                                           ^~~~~~~~
In file included from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/wtypes.h:13,
                 from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/windows.h:97:
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/combaseapi.h:232:79: error: unknown type name 'SOLE_AUTHENTICATION_SERVICE'
  232 | WINOLEAPI CoInitializeSecurity (PSECURITY_DESCRIPTOR pSecDesc, LONG cAuthSvc, SOLE_AUTHENTICATION_SERVICE *asAuthSvc, void *pReserved1, DWORD dwAuthnLevel, DWORD dwImpLevel, void *pAuthList, DWORD dwCapabilities, void *pReserved3);
      |                                                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/Users/amywe/scoop/apps/mingw/13.2.0-rt_v11-rev1/x86_64-w64-mingw32/include/combaseapi.h:257:60: error: unknown type name 'SOLE_AUTHENTICATION_SERVICE'
  257 | WINOLEAPI CoQueryAuthenticationServices (DWORD *pcAuthSvc, SOLE_AUTHENTICATION_SERVICE **asAuthSvc);
      |                                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
mingw32-make[2]: *** [CMakeFiles\ggml.dir\build.make:76: CMakeFiles/ggml.dir/ggml.c.obj] Error 1
mingw32-make[1]: *** [CMakeFiles\Makefile2:719: CMakeFiles/ggml.dir/all] Error 2
mingw32-make: *** [Makefile:145: all] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,7 +79,7 @@ if (NOT MSVC)
 endif()
 
 if (WIN32)
-    option(LLAMA_WIN_VER                     "llama: Windows Version"                           0x602)
+    set(LLAMA_WIN_VER "0x602" CACHE STRING "llama: Windows Version")
 endif()
 
 # 3rd party libs

